src/styles/rules.scss

```diff
@@ -13,30 +13,21 @@
 /// border-boxing fix
 html, body {
   font-size: $base-font-size;
-  font-weight: normal;
   line-height: $base-line-height;
-  box-sizing: border-box;
-  caret-color: $color-error;
 
-  /* Better Font Rendering =========== */
+  /* Better Font Rendering */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
 * {
   outline: none;
   outline: 0;
-  box-sizing: inherit;
 
   &:hover,
   &:focus,
   &:active {
     outline: none;
     outline: 0;
```

Comment:
Maybe it is better to implement `focus` behavior firstly?
http://www.outlinenone.com/